code_generator/kpsr_codegen/kidl_data/class_definition.py

```diff
@@ -28,9 +28,9 @@ def __init__(self, class_name, already_exists, create_builder, include_file, par
         self.enums = enums
         self.middlewares = middlewares
         self.fields = fields
-
+        self.related_classes = related_classes
```

Comment:
We just need to simply extract this and add them to the map of classes in the general code generator data.